bundles/core/org.eclipse.smarthome.core.persistence/src/main/java/org/eclipse/smarthome/core/persistence/PersistenceAdminService.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.core.persistence;
+
+import org.eclipse.smarthome.core.items.Item;
+
+/**
+ * Provides an administration interface to allow the system to purge data from the persistence store.
+ *
+ * @author Chris Jackson - Initial implementation and API
+ *
+ */
+public interface PersistenceAdminService {
+    /**
+     * Returns a list of items that are stored in the persistence service
+     *
+     * This is returned as a string to allow the persistence service to return items that are no long available as an
+     * openHAB {@link Item}.
+     *
+     * @return list of strings of item names contained in the store
```

Comment:
I think it's useful to allow maintenance of a database. I'm sure we've all created items that have had data persisted, and we've then deleted the item and created another with a different name. I think we need some way to list what is in there without reference to the item registery so it can be removed.

I'm happy to add this to the QueryablePersistenceService - my initial thought was to avoid changing the current interfaces but if you're happy that this is consolidated with the move to ESH, then I'll do that.